src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -109,6 +109,11 @@ public void addPersonKeepFilter(Person person) {
         addressBook.addPerson(person);
     }
 
+    @Override
+    public Person viewPerson(Person target) {
+        return target;
+    }
```

Comment:
Sorry, what does this method do exactly? Seems to just return the parameter, is this necessary?

Reply:
Its due to the Model interface I put a viewPerson method within it, so the ModelManager has to implement it else there would be an error